docs/justificacion_eleccion_framework_test.md

```diff
@@ -0,0 +1,39 @@
+# Elección del Test Runner :runner:
+Existen numerosas alternativas para la realización de testeo de código en proyectos de Python. Algunas de ellas son:
+
+- **Unittest** es un módulo de Python para la realización de tests. Fue diseñado para trabajar con la librería estándar de Python. Con él es posible diseñar clases para tests y definir funciones en ellas.
+- **DocTest** es un módulo incluido también en la distribución estándar de Python utilizado para los tests de caja blanca (o transparente)
```

Comment:
No, no es esto que dices, y mucho menos una biblioteca de aserciones. El objetivo de estos tests es que mostréis que entendéis los conceptos correctamente, no que copiéis cosas que os salgan como resultados de búsqueda .

Reply:
¿Se refiere a que no es un test runner?